virtual_ecosystem/core/config.py

```diff
@@ -137,15 +140,8 @@ def _resolve_config_paths(config_dir: Path, config_dict: dict[str, Any]) -> None
             file_path = Path(item)
             if not file_path.is_absolute():
                 # The resolve method is used here because it is the only method to
-                # resolve ../ entries from relative file paths. However, it also makes
-                # all paths absolute, which lengthens paths if the config directory
-                # itself is relative. The approach here converts `path/to/config` and
-                # `../data/file1.nc` to `path/to/data/file1.nc` when both paths are
-                # relative
+                # resolve ../ entries from relative file paths.
                 file_resolved = (config_dir / file_path).resolve()
```

Comment:
I would actually go all the way and use absolute paths here too, in case the path is displayed to the user at some point and it isn't clear what the path is supposed to be relative too (imagine you've put a relative path in the config file and you see a *different* relative path in the program log -- confusing). It will also be safer, if you save the path to an output file of some description and, again, it's not obvious what the path is supposed to be relative to.

```suggestion
                file_resolved = (config_dir / file_path).resolve().absolute()
```